flutter/iot_pub.dart

```diff
@@ -0,0 +1,2 @@
+final channel = ably.channels.get('device:rocket:1');
+await channel.publish('action', data: {'task': 'enable-thrusters'});
```

Comment:
Is await needed?  If this is done async, and we have no callback/action/promise, then why await?

Reply:
removed awaits 98c92021f5d6fcb4cef495127f34d0645a95a6fc